.nvmrc

```diff
@@ -1 +1 @@
-engine-strict=true
\ No newline at end of file
+v14.x
```

Comment:
It's best practice to have line break to avoid the warning you're seeing upper.

Reply:
I've corrected that, but where did you see this warning? I didn't see any warning in prettier or lint. 🤔

Reply:
<img width="914" alt="Screenshot 2021-08-09 at 16 56 33" src="https://user-images.githubusercontent.com/806/128727354-5dee4d05-f3c4-4e7c-a583-94c7a884e28c.png">

Similar to this.

Reply:
Ah thanks, corrected that. 👍🏽